frontend/talentsearch/src/js/components/signAndSubmit/SignAndSubmitPage.tsx

```diff
@@ -0,0 +1,291 @@
+      description: "Important info list item on sign and submit page.",
+    }),
+    intl.formatMessage({
+      defaultMessage:
+        "Changes made to your profile after submitting will not be updated on this copy.",
+      description: "Important info list item on sign and submit page.",
+    }),
+    intl.formatMessage({
+      defaultMessage:
+        "You are still encouraged to keep your profile up to date, updated versions will be used at later steps of the hiring process.",
+      description: "Important info list item on sign and submit page.",
+    }),
+  ];
+
+  return (
+    <ol>
+      {steps.map((item) => (
+        <li key={uniqueId()} data-h2-margin="b(bottom, m)">
+          {item}
+        </li>
+      ))}
+    </ol>
+  );
+};
+
+const Signature = ({ isNotComplete }: { isNotComplete: boolean }) => {
+  const intl = useIntl();
+  const methods = useForm<{ signature: string }>();
+  const confirmations = [
+    intl.formatMessage({
+      defaultMessage: `"I've reviewed everything written in my
+        application`,
+      description: "Signature list item on sign and submit page.",
+    }),
+    intl.formatMessage({
+      defaultMessage: `"I understand that I am part of a community who trusts each
+        other"`,
+      description: "Signature list item on sign and submit page.",
+    }),
+    intl.formatMessage({
+      defaultMessage: `"I promise that the information Ive provided is
+        true"`,
+      description: "Signature list item on sign and submit page.",
+    }),
+  ];
+  return (
+    <FormProvider {...methods}>
+      <form
+        onSubmit={methods.handleSubmit(() => {
+          // TODO: ADD SUBMIT APPLICATION MUTATION HERE
+        })}
+      >
+        <p>
+          {intl.formatMessage({
+            defaultMessage:
+              "You made it! By signing your name below you confirm that:",
+            description:
+              "Confirmation message before signature form on sign and submit page.",
+          })}
+        </p>
+        <ul>
+          {confirmations.map((item) => (
+            <li key={uniqueId()} data-h2-margin="b(bottom, m)">
+              {item}
+            </li>
+          ))}
+        </ul>
+        <div data-h2-width="b(100) s(75) m(50)" data-h2-margin="b(bottom, m)">
+          <Input
+            id="signature"
+            label={intl.formatMessage({
+              defaultMessage: "Signature",
+              description:
+                "Label displayed for signature input in sign and submit page.",
+            })}
+            type="text"
+            name="signature"
+            rules={{
+              required: intl.formatMessage(errorMessages.required),
+            }}
+            disabled={isNotComplete}
+          />
+        </div>
+        <div data-h2-text-align="b(center) s(left)">
+          <Submit
+            color="cta"
+            text={
+              <span data-h2-display="b(flex)" data-h2-align-items="b(center)">
```

Comment:
Will need to be updated to use new syntax.

Reply:
Fixed by [Update hydrogen utils and fix type error](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3678/commits/ef2a71837e961f068167149907476e257f0bc6f6)